sim/src/test_dmclock.cc

```diff
@@ -27,6 +28,13 @@
 namespace test = crimson::test_dmc;
 
 
+// Note: if this static_assert fails then our two definitions of Cost
+// do not match; change crimson::qos_simulation::Cost to match the
+// definition of crimson::dmclock::Cost.
+static_assert(std::is_same<crimson::qos_simulation::Cost,crimson::dmclock::Cost>::value,
```

Comment:
nit, could use `std::is_same_v` next time.

Reply:
Thanks; I did not know about it.